### PR TITLE
Logic for handling malware zip files

### PR DESCRIPTION
**Pull request by MaNe2020**
- zip file is extracted two times
- second extraction is using password
- after usage file is deleted

### CODE REVIEW

File: jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx [hunk 1]
```diff
@@ -0,0 +1,774 @@
...
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="unzipFileOutputFolder" elementType="Argument">
+            <stringProp name="Argument.name">unzipFileOutputFolder</stringProp>
+            <stringProp name="Argument.value">${__P(p_unzipFileOutputFolder,/path/to/the/outputfolder/)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="zipPassword" elementType="Argument">
+            <stringProp name="Argument.name">zipPassword</stringProp>
+            <stringProp name="Argument.value">${__P(p_password,infected)}</stringProp>
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx:35]: Please confirm whether this value is correct password or not. If it is it needs to be removed and the password will need to be changed

File: jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx [hunk 2]
```diff
@@ -0,0 +1,774 @@
...
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="outputPath" elementType="Argument">
+            <stringProp name="Argument.name">outputPath</stringProp>
+            <stringProp name="Argument.value">out</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="s3bucketName" elementType="Argument">
+            <stringProp name="Argument.name">s3bucketName</stringProp>
+            <stringProp name="Argument.value">${__P(p_bucket,faridpcre)}</stringProp>
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx:61]: This default value should be updated to aws-testengine-s3

File: jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx [hunk 3]
```diff
@@ -0,0 +1,774 @@
...
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="aws_service_name" elementType="Argument">
+            <stringProp name="Argument.name">aws_service_name</stringProp>
+            <stringProp name="Argument.value">${__P(p_aws_service_name,s3)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="aws_region" elementType="Argument">
+            <stringProp name="Argument.name">aws_region</stringProp>
+            <stringProp name="Argument.value">${__P(p_aws_region,us-east-1)}</stringProp>
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx:96]: This default value should be eu-west-1

**Review by jpoolGW — CHANGES_REQUESTED** [review]

**Comment by uzfarid** [comment]
@MaNe2020 plz have a look at above comments

File: jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx (hunk 1 above)
Comment (MaNe2020) [reply jmeter-icap-poc/scripts/malwareFiles/unzipDataFile.jmx:35]: @jpoolGW it is correct one. Do you want me to change it here and put some dummy value or to ask test-data team to change it to smth completely different. Also, where we are going to store this sensitive data?

**Comment by MaNe2020** [comment]
@jpoolGW and @uzfarid committed requested changes

**Pull request closed without merging**